pkg/overlay/service.go

```diff
@@ -157,26 +165,24 @@ func (s *Service) Process(ctx context.Context, _ *cobra.Command, _ []string) (
 
 	// bootstrap cache
 	var cache *Cache
-	if redisAddress != "" {
-		cache, err = NewRedisOverlayCache(redisAddress, redisPassword, db, kad)
+	if viper.GetString("redisaddress") != "" {
```

Comment:
same. what if the pointerdb has a redisaddress?

Reply:
I would argue we should never have more than one instance of redis running, across the entire app, so they should both use the same redis instance.